api/services/elastigroup/aws/paths/ecsBeanstalkImport.yaml

```diff
@@ -10,21 +10,28 @@ get:
   - "Elastigroup AWS"
   parameters:
     - $ref: "../../../../commons/parameters/accountId.yaml"
-    - $ref: "../../commons/parameters/groupId.yaml"
+    - in: path
```

Comment:
@vlad666spot I checked http://help.dev.spotinst.com/spotinst-api/elastigroup/amazon-web-services/beanstalk-api/get-beanstalk-configuration/
it seems that **groupId** is not relevant.
so it's good that you removed the ref, but why did you add it explicitly afterward? let's remove it

Reply:
fixed